config/legacy_deployment_state_migrator.go

```diff
@@ -159,6 +160,7 @@ type instance struct {
 	UUID         string `yaml:"uuid"`
 	VMCID        string `yaml:"vm_cid"`
 	DiskCID      string `yaml:"disk_cid"`
+	DiskSize     int    `yaml:"disk_size"`
```

Comment:
The disk size is used for migrating from legacy deployment to v1 in Bluemix and I just double confirmed, this is not need anymore. So I’ll remove it in next patch.